content/en/docs/concepts/security/_index.md

```diff
@@ -3,4 +3,241 @@ title: "Security"
+to providing effective cluster security.
+
+Other pages in the Kubernetes documentation have more detail about how to set up
+specific aspects of access control. The [security checklist](/docs/concepts/security/security-checklist/)
+has a set of suggested basic checks for your cluster.
+
+Beyond that, securing your cluster means implementing effective
+[authentication](/docs/concepts/security/controlling-access/#authentication) and
+[authorization](/docs/concepts/security/controlling-access/#authorization) for API access. Use [ServiceAccounts](/docs/concepts/security/service-accounts/) to
+provide and manage security identities for workloads and cluster
+components.
+
+Kubernetes uses TLS to protect API traffic; make sure to deploy the cluster using
+TLS (including for traffic between nodes and the control plane), and protect the
+encryption keys. If you use Kubernetes' own API for
+[CertificateSigningRequests](/docs/reference/access-authn-authz/certificate-signing-requests/#certificate-signing-requests),
+pay special attention to restricting misuse there.
+
+### Runtime protection: compute {#protection-runtime-compute}
+
+{{< glossary_tooltip text="Containers" term_id="container" >}} provide two
+things: isolation between different applications, and a mechanism to combine
+those isolated applications to run on the same host computer. Those two
+aspects, isolation and aggregation, mean that runtime security involves
+trade-offs and finding an appropriate balance.
+
+Kubernetes relies on a {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+to actually set up and run containers. The Kubernetes project does
+not recommend a specific container runtime and you should make sure that
+the runtime(s) that you choose meet your information security needs.
+
+To protect your compute at runtime, you can:
+
+1. Enforce [Pod security standards](/docs/concepts/security/pod-security-standards/)
+   for applications, based on the level of trust you have in them.
+1. Run a specialized operating system on your nodes that is designed specifically
+   for running containerized workloads. This is typically based on a read-only
+   operating system (_immutable image_) that provides only the services
+   essential for running containers.
+
+   Container-specific operating systems help to isolate system components and
+   present a reduced attack surface in case of a container escape.
+1. Define [ResourceQuotas](/docs/concepts/policy/resource-quotas/) to
+   fairly allocate shared resources, and use
+   mechanisms such as [LimitRanges](/docs/concepts/policy/limit-range/)
+   to ensure that Pods specify their resource requirements.
+1. Partition workloads across different nodes.
+   Use [node isolation](/docs/concepts/scheduling-eviction/assign-pod-node/#node-isolation-restriction)
+   mechanisms, either from Kubernetes itself or from the ecosystem, to ensure that
+   Pods with different trust contexts are run on separate sets of nodes.
```

Comment:
```suggestion
   the Pods with different trust contexts will run on separate sets of node.
```

Reply:
I'm afraid I won't accept this suggestion as it doesn't look like idiomatic English grammar.